framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -440,7 +440,8 @@ public void standardizeForMethodSignature(
         JavaExpressionContext context =
```

Comment:
The Javadoc of `standardizeForMethodSignature` says it is used for contract annotations, but it's not.  (I see that you added this method https://github.com/typetools/checker-framework/commit/e113bbe28f92e4f4f2d23517e995b6f14e4028df.) Is the documentation wrong or is a call to this method missing?